config/clusters/2i2c/jackeddy.values.yaml

```diff
@@ -0,0 +1,187 @@
+                mem_7_4:
+                  display_name: 7.4 GB RAM, upto 3.75 CPUs
+                  kubespawner_override:
+                    mem_guarantee: 7970807808
+                    mem_limit: 7970807808
+                    cpu_guarantee: 0.9375
+                    cpu_limit: 3.75
+                    node_selector:
+                      node.kubernetes.io/instance-type: n2-highmem-16
+                mem_14_8:
+                  display_name: 14.8 GB RAM, upto 3.75 CPUs
+                  kubespawner_override:
+                    mem_guarantee: 15941615616
+                    mem_limit: 15941615616
+                    cpu_guarantee: 1.875
+                    cpu_limit: 3.75
+                    node_selector:
+                      node.kubernetes.io/instance-type: n2-highmem-16
+                mem_29_7:
+                  display_name: 29.7 GB RAM, upto 3.75 CPUs
+                  kubespawner_override:
+                    mem_guarantee: 31883231232
+                    mem_limit: 31883231232
+                    cpu_guarantee: 3.75
+                    cpu_limit: 3.75
+                    node_selector:
+                      node.kubernetes.io/instance-type: n2-highmem-16
+                mem_60_6:
+                  display_name: 60.6 GB RAM, upto 15.72 CPUs
+                  kubespawner_override:
+                    mem_guarantee: 65105797120
+                    mem_limit: 65105797120
+                    cpu_guarantee: 7.86
+                    cpu_limit: 15.72
+                    node_selector:
+                      node.kubernetes.io/instance-type: n2-highmem-16
+                mem_121_3:
+                  display_name: 121.3 GB RAM, upto 15.72 CPUs
+                  kubespawner_override:
+                    mem_guarantee: 130211594240
+                    mem_limit: 130211594240
+                    cpu_guarantee: 15.72
+                    cpu_limit: 15.72
+                    node_selector:
+                      node.kubernetes.io/instance-type: n2-highmem-16
+            image:
+              display_name: Image
+              # https://infrastructure.2i2c.org/howto/features/allow-unlisted-profile-choice/
+              unlisted_choice:
+                enabled: True
+                display_name: "Custom image"
+                validation_regex: "^.+:.+$"
+                validation_message: "Must be a publicly available docker image, of form <image-name>:<tag>"
+                kubespawner_override:
+                  image: "{value}"
+              choices:
+                pangeo:
+                  display_name: Base Pangeo Notebook
+                  default: true
+                  slug: "pangeo"
+                  kubespawner_override:
+                    image: "pangeo/pangeo-notebook:2023.10.03"
+    hub:
+      allowNamedServers: true
+      image:
+        name: quay.io/2i2c/unlisted-choice-experiment
+        tag: "0.0.1-0.dev.git.7130.h0bdc2d30"
```

Comment:
Wondering if we should use the quay.io/2i2c/dynamic-image-building-experiment image instead to get the new UI 🤔 

@yuvipanda, what do you think?

Reply:
Currenlty the dynamic-image-building-experiment are the same as the default image, because both have modern 6.1.0 version of kubespawner with all relevant features.

Reply:
Ah, yes, you're right and we have unlisted choice with latest z2jh chart <3.

The extra in dynamic-image-building-experiment is the fancy profile I believe:
https://github.com/2i2c-org/infrastructure/blob/307bbcf00227b50b928843b9c004900cf9adf5ec/helm-charts/images/hub/dynamic-image-building-requirements.txt#L13

Reply:
@consideRatio, so you would say no to enabling the fancy spawner profile options for this hub?

Reply:
I didn't mean to opine about the use of fancy profiles - I've not caught up on that developments and the feature implications - if you think it should be enabled go for it!

I was only thinking that we should support `unlisted_choice`, which previously required a specific hub image, but it doesn't any more.

Reply:
Just to catch up, unfortunately I don't think fancy profiles should be enabled anywhere quite yet @GeorgianaElena. Hopefully later this quarter.